pkg/cmd/admin/prune/images.go

```diff
@@ -27,9 +28,19 @@ import (
 )
 
 const (
-	imagesLongDesc = `%s %s - prunes images`
 	// PruneImagesRecommendedName is the recommended command name
 	PruneImagesRecommendedName = "images"
+
+	imagesLongDesc = `Prune images no longer needed by the system due to age and status.
```

Comment:
also i think "needed by the system" isn't correct since the admin will run this:)